### ISSUE TITLE
Linux Mint 18.2 Sonya 64-bit/MATE 1.18.0. bug starting the app

### ISSUE THREAD

**Issue by marcscama** [qtlmovie/qtlmovie#10]
Bonjour,
when i try to start the program i have this bug:
QtlMovie: /usr/lib/x86_64-linux-gnu/libQt5Network.so.5: version `Qt_5' not found (required by QtlMovie)
QtlMovie: /usr/lib/x86_64-linux-gnu/libQt5Gui.so.5: version `Qt_5' not found (required by QtlMovie)
QtlMovie: /usr/lib/x86_64-linux-gnu/libQt5Multimedia.so.5: version `Qt_5' not found (required by QtlMovie)
QtlMovie: /usr/lib/x86_64-linux-gnu/libQt5Core.so.5: version `Qt_5.6' not found (required by QtlMovie)
QtlMovie: /usr/lib/x86_64-linux-gnu/libQt5Core.so.5: version `Qt_5' not found (required by QtlMovie)
QtlMovie: /usr/lib/x86_64-linux-gnu/libQt5Widgets.so.5: version `Qt_5' not found (required by QtlMovie)

i have all this files  installeted in the correct position but the program doesn't find them.

2) this program is good for ripping a dvd in loseless mode, like for example makemvk?
you can add this request/option for example using dvdbackup app?

3) do you a now a method under linux/ubuntu to convert some vobs files into 1 single mkv file?

thanks
bye

**Comment by lelegard** [comment]
Hi,

1. I do not know Linux Mint, so I cannot comment. I know and use Fedora, Ubuntu and RHEL/CentOS. I have created .rpm and .deb packages for them. But I do not know how the Qt environment is deployed on Mint. Apparently, you managed to build QtlMovie. What you see does not seem specific to QtlMovie and has been probably already experienced with other Qt applications on Mint. I can only advise to follow that track.

2. QtlMovie can indeed rip DVD's in lossless mode, like many other applications. You can extract the ISO image, the tracks or the files. Or directly convert for iPad or whatever.

3. Take care with VOB's. There look like .mpg files but they are not. A set of VOB files may sometimes contains several mixed video on the same track, the Program Chains (PGC). The "DVD extraction" tool in QtlMovie shows the number of PGC and angles. When there is only one PGC/angle, as it is quite often the case, the VOB is like a .mpg file and any video converter will do the job. Otherwise, you have to demux the PGC you want. QtlMovie is aware of PGC and demuxes the one you select. Well, at the beginning it didn't, until I got a completely messed up video file from "The Rocky Horror Picture Show" DVD. At that time, I realised the PGC issue and added PGC demux in QtlMovie.

**Comment by marcscama** [comment]
bonjour,
thank you very much for your message!
 linux mint it is based on ubuntu xenial. i try myself to find  a solution.

the program that i need in this moment is something like makemkv: a program that not transcode.
 i use handbrake but this one convert everithing.
Qtlmovie you say that rip a dvd but with transcoding or not? Is it possible to keep the audio and video without convert in other format?
 thank and goodday

**Comment by lelegard** [comment]
> Qtlmovie you say that rip a dvd but with transcoding or not? Is it possible to keep the audio and video without convert in other format?

Yes, the "DVD Extraction" tool ("Tools" menu in QtlMovie) is just a ripper, it extracts the bit-to-bit content, either as an ISO image or individual files (including all VOB files that you can later repackage or transcode).

**Comment by Darkle** [comment]
I ran in to this issue too on Ubuntu 16.04. My workaround was this:

1. Download the Qt installer from https://www.qt.io/download and install Qt version 5.6
2. Create a bash script with the following: 

	```
	#!/bin/bash
	export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/home/coop/Qt/5.6/gcc_64/lib/
	QtlMovie
	```
	(replace the `/home/coop/Qt/5.6/gcc_64/lib/` with wherever you installed Qt.

3. Create another QtlMovie.desktop file (with an altered Name key value, e.g. "QtlMovie (Qt fixed)" and point it to the bash script.
